### DISCUSSION TITLE
How can I help translate Pimcore?

### DISCUSSION THREAD

**Discussion by blankse** [pimcore/pimcore#16778] [Q&A / Support] [answered]
In the readme is a sentence:

> Help translate Pimcore! Start with [Essentials](https://poeditor.com/join/project/VWmZyvFVMH), continue with [Extended](https://poeditor.com/join/project/XliCYYgILb)

But there are no languages in the POEditor project:
![image](https://github.com/pimcore/pimcore/assets/998558/6e197d7b-1b12-4a02-ba4b-576b7ab6a547)

**Comment by blankse** [comment]
@brusch @fashxp @kingjia90 Are translations from the community no longer desired?

**Reply by kingjia90** [reply]
Thank you for reporting!

That's surely not the case, maybe the invitation link expired or something like that, i believe only @brusch has access to it so far, so we have to wait for him to check it out

**Reply by blankse** [reply]
@kingjia90 Can you tell him differently? I don't think he reacts to mentions.

**Reply by kingjia90** [reply]
Sure, i have left a direct message, we were quite busy due to the Inspire, there might be some queue, speaking of that, congratulations on being MVP 2 times in a row! Well also for soon or already, the Most valuable Pappa (papa,pa,pater) for 2 times 😄

**Reply by jdreesen** [reply]
Oh wow, congratulations @blankse! Well deserved! :tada:

**Reply by brusch** [reply, marked as answer]
@blankse thanks for reporting, unfortunately those 2 projects were automatically archived by POEditor. 
Just restored them, working again 😊

**Reply by blankse** [reply]
@kingjia90 @jdreesen Thank you for the congratulations. The baby is not here yet.
@brusch Thank you for fixing.

**Comment by vmalyk** [comment]
Hi! 

@brusch @fashxp @kingjia90 looks like translations projects has been archived again - no languages in POEditor.

**Reply by brusch** [reply]
@vmalyk yep, just restored them, thanks for the hint 👍😊
